src/sfizz/Region.cpp

```diff
@@ -932,11 +814,49 @@ bool sfz::Region::parseOpcode(const Opcode& rawOpcode)
         break;
     default:
         return false;
+
+    #undef case_any_ccN
     }
 
     return true;
 }
 
+bool sfz::Region::processGenericCc(const Opcode& opcode, Range<float> range, CCMap<Modifier> *ccMap)
```

Comment:
ccMap could be a reference maybe? Do you expect that this function could be called for a null map?

Reply:
range could also be const ref although it's a small object anyway.

Reply:
I don't expect these CCMaps to remain as this long-term. I wish to move it somewhere related with the mod matrix. They stay around only for now to preserve unit tests.

As of the const-ref, Range is a small enough object that I pass it like this.
Range=2*32 bits no indirection, Ref=64 bits and indirection